src/localtunnel.ts

```diff
@@ -12,14 +15,37 @@ export interface ITunnelConfigs {
   subdomain?: string,
 }
 
+export async function getJsonPath(context: ExtensionContext) {
+  const workspaces = vscode.workspace.workspaceFolders;
+  console.info('workspaces', workspaces);
+
+  if (workspaces == null) return '';
+
+  const workspacePath = workspaces[0].uri.fsPath;
+  const jsonPath = workspacePath + '/.vscode/localtunnel.json';
+  return jsonPath;
+}
+
 export async function start(context: ExtensionContext) {
   const currentTunnel = context.workspaceState.get('currentTunnel') as ITunnel;
   if (currentTunnel) {
     return window.showErrorMessage('Current Tunnel is already running, please stop it before starting another tunnel');
   }
 
-  const port = await getPort();
-  const subdomain = await getSubdomain();
+  let subdomain;
+  let port;
+  const jsonPath = await getJsonPath(context);
+
+  if (fs.existsSync(jsonPath)) {
```

Comment:
This condition means that if a user has a default configuration for localtunnel in a .vscode/localtunnel.json for that workspace, they'll never be able to manually add the port and subdomain using the vs code input right? so they never really reach the `else` condition?

If that's the case, we should before reaching this if, ask if the user wants to grab the configuration from the workspace configuration, or if they want to input it manually.